src/ui/element_download_list.cc

```diff
@@ -67,40 +31,42 @@ ElementDownloadList::ElementDownloadList() :
+                                                                                 "{d.connection_seed.set=initial_seed, print=\"Enabled initial seeding for the selected download.\"}");
+
+  m_bindings['U'] = std::bind(&ElementDownloadList::receive_command, this, "d.delete_tied=; print=\"Cleared tied to file association for the selected download.\"");
 
   // These should also be commands.
-  m_bindings['1']           = std::bind(&ElementDownloadList::receive_change_view, this, "main");
-  m_bindings['2']           = std::bind(&ElementDownloadList::receive_change_view, this, "name");
-  m_bindings['3']           = std::bind(&ElementDownloadList::receive_change_view, this, "started");
-  m_bindings['4']           = std::bind(&ElementDownloadList::receive_change_view, this, "stopped");
-  m_bindings['5']           = std::bind(&ElementDownloadList::receive_change_view, this, "complete");
-  m_bindings['6']           = std::bind(&ElementDownloadList::receive_change_view, this, "incomplete");
-  m_bindings['7']           = std::bind(&ElementDownloadList::receive_change_view, this, "hashing");
-  m_bindings['8']           = std::bind(&ElementDownloadList::receive_change_view, this, "seeding");
-  m_bindings['9']           = std::bind(&ElementDownloadList::receive_change_view, this, "leeching");
-  m_bindings['0']           = std::bind(&ElementDownloadList::receive_change_view, this, "active");
-
-  m_bindings[KEY_UP]   = m_bindings['P' - '@'] = std::bind(&ElementDownloadList::receive_prev, this);
+  m_bindings['1'] = std::bind(&ElementDownloadList::receive_change_view, this, "main");
+  m_bindings['2'] = std::bind(&ElementDownloadList::receive_change_view, this, "name");
+  m_bindings['3'] = std::bind(&ElementDownloadList::receive_change_view, this, "started");
+  m_bindings['4'] = std::bind(&ElementDownloadList::receive_change_view, this, "stopped");
+  m_bindings['5'] = std::bind(&ElementDownloadList::receive_change_view, this, "complete");
+  m_bindings['6'] = std::bind(&ElementDownloadList::receive_change_view, this, "incomplete");
+  m_bindings['7'] = std::bind(&ElementDownloadList::receive_change_view, this, "hashing");
+  m_bindings['8'] = std::bind(&ElementDownloadList::receive_change_view, this, "seeding");
+  m_bindings['9'] = std::bind(&ElementDownloadList::receive_change_view, this, "leeching");
+  m_bindings['0'] = std::bind(&ElementDownloadList::receive_change_view, this, "active");
+
+  m_bindings[KEY_UP] = m_bindings['P' - '@'] = std::bind(&ElementDownloadList::receive_prev, this);
   m_bindings[KEY_DOWN] = m_bindings['N' - '@'] = std::bind(&ElementDownloadList::receive_next, this);
 
-  m_bindings['L']           = std::bind(&ElementDownloadList::toggle_layout, this);
+  m_bindings[KEY_PPAGE] = [this] { receive_prev_page(); };
+  m_bindings[KEY_NPAGE] = [this] { receive_next_page(); };
+
+  m_bindings[KEY_HOME] = [this] { receive_home(); };
+  m_bindings[KEY_END]  = [this] { receive_end(); };
```

Comment:
Include bindings for these; ^P, ^N, and something for home/end.

Reply:
^P and ^N are taken for up/down, and I couldn't find a good consensus, so went with ^J and ^K.

I used ^A and ^E for home/end, that's pretty standard (e.g. readline/emacs)